### ISSUE TITLE
Formalize the problem statement

### ISSUE THREAD

**Issue by agitter** [gitter-lab/ml-bio-workshop#46]
We would like to formally articulate what problem this workshop solves in a few sentences. The [GLBIO abstract](https://www.iscb.org/glbio2019-program/glbio2019-abstracts#education2) gives some ideas.

The overall concept we discussed during out last meeting is that we want to demystify machine learning and artificial intelligence concepts and help raise awareness of when good opportunities for classification arise during biological research.

**Comment by cmilica** [comment]
Here are a few things from the abstract that might help
- participants are often taught to mechanically run through a software pipeline for certain tasks **without learning the best practices** in various stages of the workflow.

- the issue might be **traditional task-centric approaches**, without using the proper mindset when it comes to applying machine learning in the biology research and the **lack of ability to critically analyze machine learning applications in their domain**.

- the community clearly has a **need to explore novel educational frameworks** in order to address these challenges in teaching machine learning to biologists.

- ml4bio workshop prioritizes **teaching machine learning literacy**, that is, the right way to set up learning problems, how to reason about learning algorithms, and how to assess learned models. 

- **the holistic view of the machine learning workflow** vs. immediately dive into the technicalities of coding and mathematics.

**Comment by cmilica** [comment]
Should we close this issue?

**Comment by agitter** [comment]
We could also review the first introductory paragraph in the website to see whether it should be more explicit about the problem the workshop solves.  Currently:
> The Machine Learning for Biology (ml4bio) workshop is aimed at biologists with no previous machine learning experience. This is a conceptual workshop designed to teach machine learning literacy. After the workshop, the participants will be able to define machine learning concepts like samples, features, training set, validation set, test set, evaluation metrics, and model selection. The participants will be able to interactively develop an understanding of machine learning classifiers commonly used in biology like decision trees, random forests, logistic regression, and neural networks. The focus will be on problems in biology where machine learning is effectively used.


I'm okay closing this if you think it looks good.

**Comment by dltreu** [comment]
This looks good!


Debora

________________________________

**Comment by cmilica** [comment]
Looks good to me too!

**Comment by agitter** [comment]
Great, I'll close the issue.
